### PR TITLE
feat: make settings accordion click area larger

### PR DESCRIPTION
**Pull request by Ploonet**
# Description

Changes the setting-accordion Svelte component to make the click area larger.

Currently the only clickable area to open the accordion is the arrow to the right side.

This PR extends that area to contain the title, description and the space between the text and the arrow.

## Why

I believe this makes clicking the items easier, which helps accessibility on large high res displays and on mobile in my opinion.

## Other info

- This is my first open source PR ever, i hope i'm doing things right
- I've had issues running the dev version locally (couldn't get past the admin registration), so i was not able to test directly on the Settings pages. I built this by creating a SettingAccordion in the registration page.

### CODE REVIEW

**Review by bo0tzz — APPROVED** [review]
I'd been meaning to add this, thank you for the PR! I tested in the admin page and it works as expected. Can you fix the broken checks/tests?

**Comment by Ploonet** [comment]
> I'd been meaning to add this, thank you for the PR! I tested in the admin page and it works as expected. Can you fix the broken checks/tests?

Great ! Thanks for approving. :) 

The tests & checks are ok now i think.

**Comment by Ploonet** [comment]
If you need me to rebase my 2 commits into 1 before merging, just tell me !

**Comment by alextran1502** [comment]
@Ploonet No problem, thank you, we squash and merge all PR.

File: web/src/lib/components/admin-page/settings/setting-accordion.svelte [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </script>
 
 <div class="border-b-[1px] border-gray-200 py-4 dark:border-gray-700">
-  <div class="flex place-items-center justify-between">
+  <button on:click={toggle} class="flex cursor-pointer place-items-center justify-between text-left">
```
Comment (jrasm91) [inline web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: Button doesn't need cursor-pointer btw

Comment (Ploonet) [reply web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: Ah yes indeed.. It was a div in my initial commit, I forgot to remove it afterwards.

Should I make a new PR?

Comment (jrasm91) [reply web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: Not a big deal.. would not hurt to fix it either.

Comment (bo0tzz) [reply web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: It seems like the changes after my review also did something that caused the arrow button to move to the left. Wouldn't hurt to fix that either.
image

You mentioned you had trouble with your dev env - if you want some help fixing that don't hesitate to hop into Discord.

Comment (jrasm91) [reply web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: You can try this too

https://immich.app/docs/developer/setup#connect-to-a-remote-backend

Comment (Ploonet) [reply web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: Sorry about the issue caused by my commit. I've fixed it here if you'd like to take a look : #4129 

jrasm the remote backend works much better than my local env, thanks. :) I'll post a message on discord to get help if i get stuck again !

Comment (bo0tzz) [reply web/src/lib/components/admin-page/settings/setting-accordion.svelte:11]: Thank you!

**Pull request merged**